Packs/fireeye/Integrations/Fireeye/Fireeye.js

```diff
@@ -369,6 +369,15 @@ if (currentCommand.extended) {
         entry.EntryContext[current.contextPath] = context;
 
         if(command === 'fe-submit-result' || command === 'fe-submit-url-result'){
+            if (command === 'fe-submit-result' && 'submissionStatus' in result && result.submissionStatus.submissionStatus === 'In Progress'){
```

Comment:
if result is not a dict, but class, the in operator should not work to check if an attribute exists. We should use hasattr